library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -42,7 +43,19 @@ internal class TransactionPayloadFragment :
     Fragment(), SearchView.OnQueryTextListener {
 
     private lateinit var payloadBinding: ChuckerFragmentTransactionPayloadBinding
-    private val payloadAdapter = TransactionBodyAdapter()
+
+    private val payloadAdapter = TransactionPayloadAdapter()
+
+    private val headerAdapter by lazy { TransactionHeadersAdapter() }
+    private val bodyAdapter by lazy { TransactionBodyAdapter() }
+    private val imageAdapter by lazy { TransactionImageAdapter() }
```

Comment:
These 3 adapter are not yet called, is it normal?

Reply:
It is a draft PR, it is not finalized as well as not meant for review.